<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteWidget</class>
 <widget class="QWidget" name="NoteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>2</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="headingAndDeleteButtonLayout">
       <item>
        <widget class="QLineEdit" name="headingEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string>X</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="verticalMoveButtonsAndBodyLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalMoveButtonsLayout">
         <item>
          <widget class="QPushButton" name="moveUpButton">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>↑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveDownButton">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>↓</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="bodyEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalMoveButtonsLayout">
       <item>
        <widget class="QPushButton" name="moveLeftButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>←</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveRightButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>→</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>headingEdit</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>bodyEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
